src/main/scala/uk/gov/nationalarchives/tdr/api/http/GraphQLServer.scala

```diff
@@ -59,18 +59,24 @@ object GraphQLServer {
     val uuidSourceClass: Class[_] = Class.forName(ConfigFactory.load().getString("source.uuid"))
     val uuidSource: UUIDSource = uuidSourceClass.getDeclaredConstructor().newInstance().asInstanceOf[UUIDSource]
     val db = DbConnection.db
+
+    val consignmentRepository = new ConsignmentRepository(db)
```

Comment:
Is there a reason why the consignment repo object has been assigned to a val, unlike the other repos which are defined in the service constructors?

Reply:
It's because it's being used more than once - it's passed to the ConsignmentService and the FileService.

I couldn't find a neater way to do it, but I thought it was OK on balance - some of the services might need to fetch data from different repositories in order to keep some business logic in the service layer, so it's not cluttering the the GraphQL layer. What do you think?

Reply:
OK. Missed it being used more than once. Seems like a reasonable approach.